src/pages/HomeNeedLeadView.tsx

```diff
@@ -0,0 +1,254 @@
+import React from "react";
+import { makeStyles, Box } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import doctorMaskImage from "../images/homepage_doctor_mask.jpg";
+// import { GET_NEEDS } from "../service/apollo/queries";
+// import { useQuery } from "@apollo/react-hooks";
+
+const useStyles = makeStyles(() => ({
+  box: {
+    marginTop: "50px",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+    background: `url(${doctorMaskImage}) no-repeat center center `,
+    backgroundSize: "cover",
+    height: "100vh"
+  },
+  box_table: {
+    height: "45vh",
+    width: "100%",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: "rgba(255, 255, 255, 0.8)",
+    color: "#293d6c",
+    paddingBottom: "25px",
+    minWidth: "415px",
+    minHeight: "280px"
+  },
+  table: {
+    width: "80%"
+  },
+  button: {
+    fontWeight: 400,
+    width: "12%",
+    margin: "20px 0px 10px 0px"
+  },
+  header: {
+    textAlign: "start",
+    fontWeight: 500,
+    padding: "5px 0px"
+  },
+  tr: {
+    color: "#6a6a6a",
+    fontWeight: 400,
+    marginTop: "20px"
+  },
+  td: {
+    padding: "2px"
+  },
+  h3: {
+    fontSize: "18px"
+  },
+  link: {
+    color: "#293d6c"
+  }
+}));
+
+const HomeNeedLeadView = () => {
+  //   const { data: needsData } = useQuery(GET_NEEDS, {
+  //     variables: {
+  //       request: {
+  //         options: {
+  //           limit: 5
+  //         }
+  //       }
+  //     },
+  //     fetchPolicy: "network-only"
+  //   });
+  //   const needs = needsData?.getAllNeeds?.needs;
+
+  const styles = useStyles();
+
+  return (
+    <>
+      <Box className={styles.box}>
+        <Box className={styles.box_table}>
+          <h3 className={styles.h3}>VIEW NEEDS</h3>
+          <table className={styles.table}>
+            <thead>
+              <tr>
+                <th className={styles.header}>URGENCY</th>
+                <th className={styles.header}>NEED</th>
+                <th className={styles.header}>QTY</th>
+                <th className={styles.header}>DONATE/BUY</th>
+                <th className={styles.header}>CITY</th>
+                <th className={styles.header}>NAME</th>
+                <th className={styles.header}></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr className={styles.tr}>
```

Comment:
why have you not implemented the query ?  You just create a json file with mock data otherwise, so you can just create a map() to diplay the data here